docs/source/getting_started/quickstart.rst

```diff
@@ -112,6 +112,8 @@ Qwen2-7B-Instruct as an example:
 
    python -m vllm.entrypoints.openai.api_server --model Qwen/Qwen2-7B-Instruct
 
+.. tip:: If saw connection to https://huggingface.co/ failed, try: ``HF_ENDPOINT=https://hf-mirror.com python -m vllm.entrypoints.openai.api_server --model Qwen/Qwen1.5-7B-Chat``.
```

Comment:
Enhancement Suggestions

1. Specify Error Handling
Clarify when to use the mirror site by detailing specific error messages.

2. Automate the Mirror Switch
Provide a script that detects connection issues and automatically switches to the mirror site.

3. Set Environment Variables Permanently
Recommend setting the HF_ENDPOINT in the user’s shell configuration to simplify commands.

4. Provide Comprehensive Error Documentation
Offer detailed guidance on diagnosing connection issues, including how to access error logs and troubleshooting steps.